docs/.aicoe-ci.yaml

```diff
@@ -10,6 +10,7 @@ build:
   base-image: registry.access.redhat.com/ubi8/ubi:latest
   build-stratergy: Dockerfile # Allowed values: Source, Dockerfile, Containerfile (default: Source)
   dockerfile-path: Dockerfile
+  custom_tag: latest # custom tag to be push to registry
```

Comment:
@harshad16 Is the key wrong? Should it be `custom-tag` to match all the other keys?

Reply:
ah , good eyes :) 
will change this along with the documentation updates.